chain/rosetta-rpc/Cargo.toml

```diff
@@ -22,7 +22,7 @@ serde = { version = "1", features = ["derive"] }
 serde_json = "1"
 validator = "0.10"
 
-paperclip = { version = "0.4", features = ["actix", "actix-nightly"] }
+paperclip = { version = "0.4", features = ["actix"] }
```

Comment:
is it still compilable with `rosetta_rpc` feature enabled? (the feature is disabled by default)

```
cargo check --features rosetta_rpc
```

Reply:
Yep, do you know what was needed by `actix-nightly`?

Reply:
I believe I was getting some cryptic errors and I was trying to use nightly features to get better error messages from macros. If it is compilable without it, it is fine to drop it :+1: